### PR TITLE
Feature/uppsf 3431 use new ontology structures in handlers

### PR DESCRIPTION
**Pull request by ManoelMilchev**
# Description

## What

Migrate the concepts-rw-neo4j handlers to the new ontology structures, update all of the integration tests.

## Why

[JIRA](https://financialtimes.atlassian.net/browse/UPPSF-3431)

## Scope and particulars of this PR (Please tick all that apply)

- [ ] Tech hygiene (dependency updating & other tech debt)
- [ ] Bug fix
- [x] Feature
- [ ] Documentation
- [ ] Breaking change
- [ ] Minor change (e.g. fixing a typo, adding config)

___
This Pull Request follows the rules described in our [Pull Requests Guide](https://github.com/Financial-Times/upp-docs/tree/master/guides/pr-guide)

### CODE REVIEW

**Review by ivan-p-nikolov — APPROVED** [review]

File: concepts/concepts_service_test.go [hunk 1]
```diff
@@ -1042,34 +1092,71 @@ func TestWriteMemberships_CleansUpExisting(t *testing.T) {
...
+func extractFieldFromRelationship(r ontology.Relationships, relationshipName string) string {
+	for i := range r {
+		if r[i].Label == relationshipName {
+			return r[i].UUID
+		}
+	}
+	return ""
 }
 
 func TestWriteMemberships_FixOldData(t *testing.T) {
```
Comment (ivanruski) [inline concepts/concepts_service_test.go:1162]: **question:** @ivan-p-nikolov If I am not mistaken, we don't use transform package in aggy anymore. If we remove this test, `concepts-rw-neo4j` won't depend on the `transform` pkg and soon we might be able to deprecate it ... if we want such a thing.

**Review by ivanruski — APPROVED** [review]

File: concepts/concepts_service_test.go (hunk 1 above)
Comment (ivan-p-nikolov) [reply concepts/concepts_service_test.go:1162]: Yes, in theory. But that would mean rewriting all old tests.

Comment (ivanruski) [reply concepts/concepts_service_test.go:1162]: Okay, maybe in aggy there is more, but here this is the only test that uses the `transform` package.

Comment (ivanruski) [reply concepts/concepts_service_test.go:1162]: It's not relevant to this PR, I was just asking.

**Pull request merged**